pytorch/src/DFT.cxx

```diff
@@ -0,0 +1,124 @@
+#include "WireCellPytorch/DFT.h"
+#include "WireCellUtil/NamedFactory.h"
+
+#include <torch/script.h>
+#include <torch/csrc/api/include/torch/fft.h>
```

Comment:
seems [this file](https://github.com/pytorch/pytorch/commits/master/torch/csrc/api/include/torch/fft.h) is not available for older libtorch versions, e.g. 1.6 which is the latest version on gpvm.

Reply:
Hmm, this takes me by surprise.

Fermilab should upgrade.  IIRC, they also do not build libtorch with CUDA support which also limits their usefulness.

I am inclined to say that we simply do not include WCT's `pytorch` sub-package for the `wirecell` UPS product until they upgrade.  To avoid a build-time error in the case that libtorch is found automatically would rquire adding `--with-libtorch=no` to `wcb configure`.  My own intention is to use this new torch support in a stand-alone build of WCT so I don't care about this UPS breakage.  

An innocent bystander of that choice would be that dnnroi will also not be available in a UPS environment until Fermilab upgrades.  If we "hide" the libtorch based DFT behind some `#ifdef` then at least dnnroi could run on CPU at Fermilab.  But, I don't know if doing that is on any immediate goal (is it?).

What's your opinion on either of these approaches, @HaiwangYu ?  Any other solution?

Reply:
Seems this `fft.h` is available from 1.7. https://github.com/pytorch/pytorch/tree/v1.6.0/torch/csrc/api/include/torch
And the versioning macros are available from 1.8 https://pytorch.org/cppdocs/notes/versioning.html
I think we can skip the `pytorch` folder until gpvm has a newer version. Before that, if DNNROI is needed, we can remove `DFT.cxx` source file in the building shims. How about this? @brettviren

Reply:
Sounds very good to me.  Thanks.